docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+# Why is this PR needed?
+
+# What does the PR do?
+
+# Does it break backwards compatibility?
```

Comment:
We could make this a checkbox to be fancy. . . I don't know how though :)

Reply:
I could easily find out, if you think it improves the workflow.

Would it still makes sense though if we sometimes want to put an extra remark such as these: https://github.com/graknlabs/grakn/pull/2542